src/main/resources/config/liquibase/akip-master.xml

```diff
@@ -44,8 +44,8 @@
     <include file="config/liquibase/changelog/00000000002001_added_entity_AkipEmailConnectorConfig.xml" relativeToChangelogFile="false"/>
 
     <include file="config/liquibase/changelog/00000000003001_added_entity_AkipPromptConfiguration.xml" relativeToChangelogFile="false"/>
-
-
-
+    <include file="config/liquibase/changelog/00000000000020_added_entity_TaskDefinition.xml" relativeToChangelogFile="false"/>
```

Comment:
Start the changesets to 019 instead of 020 and move them up. Just below the changeset 018. Move the constraint changeset to below the changeset 009.

Reply:
There is already a changeset with id 20, it's just not in the master, that's why I put 20